features/vaccineData.js

```diff
@@ -0,0 +1,28 @@
+const vaccine = require("../repositories/vaccineOps.js");
+
+//Should Take in vaccine keyword and two letter state
+module.exports = function (app) {
+  //Regular Expression search (vaccine + State Abbreviation)
+  let vaccineSearch =
```

Comment:
we should make this a constant and probably pull it out of this function, perhaps on line 2.  we could create a constants file later to help organize this, but we don't need to do that now.